### ISSUE TITLE
Simple theme: Web Accessibility

### ISSUE THREAD

**Issue by dalf** [searxng/searxng#818]
The simple theme should implement the ARIA recommendations.

Here some items & links that can help to the basic steps:
* https://developer.mozilla.org/en-US/docs/Tools/Accessibility_inspector
* https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/
* on a google.com result page, press tab various accessibility links including "skip to content": https://css-tricks.com/how-to-create-a-skip-to-content-link/
* the preferences, engine tab should have a "disable all engines" / "enable all engines" (in the tab, not globally)

----

Some of the issues:
* [ ] The tabs, the categories and the checkboxes (in the preferences) can't get the focus ( [`display: none`](https://github.com/searxng/searxng/blob/a4942213a1dc6404616e239e0d2fb55a6e19a499/searx/static/themes/simple/src/less/search.less#L33) must be replaced by [`appearance: none`](https://developer.mozilla.org/en-US/docs/Web/CSS/appearance)):
   * [x] checkbox: https://codepen.io/dalf-the-encoder/pen/VwrQKKm
   * [ ] tabs 
* [x] #1316

**Comment by dalf** [comment]
Some experiment with the checkbox: https://codepen.io/dalf-the-encoder/pen/VwrQKKm?editors=1100 
* the checkbox can get the focus (it is not possible currently because of [`visibility: hidden;`](https://github.com/searxng/searxng/blob/12b5f9e5611d40d80de7efdfc1fd6afae6c8cdaf/searx/static/themes/simple/src/less/toolkit.less#L405))
* the focus is shown
* the checkbox can be disabled
* it is possible to click anywhere to uncheck/check (the circle or the horizontal bar).
* there is a checkmark when the checkbox is checked (perhaps useless)

![image](https://user-images.githubusercontent.com/1594191/154773774-9f794521-e3ea-4cdc-9d4b-c9a2cba1ff0d.png)

... may be the usual checkboxes are better (but styled with CSS so it is not browser dependent).

What are your opinions?
